packages/scandipwa/src/component/ProductTabs/ProductTabs.component.tsx

```diff
@@ -48,6 +60,16 @@ export class ProductTabsComponent extends PureComponent<ProductTabsComponentProp
         }
     }
 
+    handleResize(entries: ResizeObserverEntry[]): void {
+        if (this.tabContentRef.current) {
+            this.tabContentRef.current.style.height = `${entries[0].contentRect.height }px`;
```

Comment:
probably better to use `CSS` util from the core.
something like this should work probably: `CSS.setVariable(this.tabContentRef, 'tab-content-height, ``${newHeight}px``);`
this would update the `tab-content-height` variable, which then you can use in the style file like: `var(--tab-content-height)`.

Reply:
also, better to destructure `entries[0]` and name it something more descriptive.